src/appengine/handlers/jobs.py

```diff
@@ -127,6 +131,7 @@ def post(self):
     job.description = description
     job.environment_string = environment_string
     job.templates = templates
+    job.project = project
```

Comment:
How about setting this as a `_pre_put_hook` in data_types.Job? That will be a bit more reliable, and can prevent issues in the future when we write e.g. migration scripts.

Reply:
Great idea, forgot about hook. done!

Reply:
this line needs to be removed now?

Reply:
yes fixed after last patch